integrationTests/testProcessorNode.go

```diff
@@ -2888,7 +2893,7 @@ func (tpn *TestProcessorNode) createHeartbeatWithHardforkTrigger() {
 
 	// TODO: remove it with heartbeat v1 cleanup
 	// =============== Heartbeat ============== //
-	redundancyHandler := &mock.RedundancyHandlerStub{}
+	/*redundancyHandler := &mock.RedundancyHandlerStub{}
```

Comment:
commented code?

Reply:
my bad, forgot it while testing heartbeat v2 only..fixed it